adding_species_manuscript.tex

```diff
@@ -490,7 +488,7 @@ \subsection*{Basic setup for chromosome-level simulations}
   However, we anticipate future development will provide support for more complex, heterogeneous mutational processes,
   as these are easily specified in both the \texttt{SLiM} and \texttt{msprime} simulation engines.
   Such progress will further improve the realism of simulated genomes,
-  since mutation rates and processes are known to vary along the genome and through time \citep{Benzer1961,Ellegren2003,Supek2019}.
+  since mutation rates and evolutionary processes are known to vary along the genome and through time \citep{Benzer1961,Ellegren2003,Supek2019}.
 
```

Comment:
The intent here is mutation processes and not just any processes. I seem to recall that @petrelharp added this, so I defer this to him.

Reply:
I guess that 'mutation rates and mutation processes' makes sense.

Reply:
Another possibility that may feel less syntactically awkward, "mutation processes, including rates, are known to vary..."

Reply:
I like Elise's suggestion